src/models/GlobalStyle.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import type { RuleSet, Stringifier } from '../types'
+import flatten from '../utils/flatten'
+import isStaticRules from '../utils/isStaticRules'
+import StyleSheet from './StyleSheet'
+
+export default (stringifyRules: Stringifier) => {
+  class GlobalStyle {
+    rules: RuleSet
+    componentId: string
+    isStatic: boolean
+
+    constructor(rules: RuleSet, componentId: string) {
+      this.rules = rules
+      this.componentId = componentId
+      this.isStatic = isStaticRules(rules)
+      if (!StyleSheet.master.hasId(componentId)) {
+        const placeholder =
+          process.env.NODE_ENV !== 'production' ? [`.${componentId} {}`] : []
```

Comment:
I think you don’t need this “marker class”. Since the global styles won’t be referring to it it’ll be pretty useless 😅

Reply:
Oops!  😅